wowup-electron/src/app/services/warcraft/warcraft-installation.service.ts

```diff
@@ -228,7 +238,8 @@ export class WarcraftInstallationService {
       const typeName = getEnumName(WowClientType, product.clientType);
       const currentInstallations = await this.getWowInstallationsByClientType(product.clientType);
 
-      const label = await this.getNewInstallLabel(typeName, currentInstallations.length);
+      const label = currentInstallations.length == 0 ? null :
+        await this.getNewInstallLabel(typeName, currentInstallations.length);
```

Comment:
If you make `getNewInstallLabel` return `null` if the length is 0, this change and the one below can be reverted